### ISSUE TITLE
Error in "Network information" of Event ID 4648

### ISSUE THREAD

**Issue by NicknameNotTaken** [MicrosoftDocs/windows-itpro-docs#10771]
Hi,

There seems to be a mistake in the "Network information" part of the documentation for the Event ID 4648. It says that the IpAddress and Port are related to the source/client host. However, unless i misunderstand something, what is logged here is the informaton regarding the remote host. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 19e21588-1388-f702-e571-24c547cf967d
* Version Independent ID: a0f454cc-e44d-2285-aaa1-f4bbc2c4c8de
* Content: [4648(S) A logon was attempted using explicit credentials. (Windows 10) - Windows security](https://docs.microsoft.com/en-us/windows/security/threat-protection/auditing/event-4648)
* Content Source: [windows/security/threat-protection/auditing/event-4648.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/security/threat-protection/auditing/event-4648.md)
* Product: **m365-security**
* Technology: **windows-sec**
* GitHub Login: @Dansimp
* Microsoft Alias: **dansimp**

**Comment by RandomUser1215R** [comment]
Hello, 
any news on this ?

**Comment by yogkumgit** [comment]
@Dansimp Please help us in resolving this issue. Thanks

**Comment by yogkumgit** [comment]
@vinaypamnani-msft Please help us in resolving this issue. Thanks

**Comment by NicknameNotTaken** [comment]
Hi, 
I just saw the tag "writer-input-required". Do you need additionnal information?

**Comment by yogkumgit** [comment]
@NicknameNotTaken No, This is a request to Vinay to verify the error and suggest if the document update is required or not. Thanks
